assets/css/styles.css

```diff
@@ -160,22 +220,153 @@ h2 {
 ::placeholder {
     text-align: center;
     font-style: normal;
+    color: #1c7aa7;
 }
 
 .form_data {
-    border: 1px solid black;
+    border: 1px solid rgba(209, 223, 232, 0.5);
     height: 30px;
     width: 200px;
-    background-color: #cde3f7;
+    background-color: #fff;
+    border-radius: 3px;
 }
 
 .form_info {
     margin-top: 30px;
+    margin-bottom: 1px;
 }
 .form_text {
     font-size: 15px;
+    color: #fff;
+    font-weight: bold;
+    text-decoration: underline;
+}
+
+.features {
+    /*background-color: rgba(28,122,167,0.4);*/
+    padding: 100px 0;
+}
+
+.features h2 {
+    color: #1c7aa7;
+}
+
+.feature-row:nth-of-type(2) {
+    margin-top: 50px;
+}
+
+.feature-icon-container {
+    text-align: center;
+    font-size: 40px;
+    margin-top: 40px;
+    color: #1c7aa7;
+}
+
+.container__team {
+    padding: 65px 120px;
+}
+
+.team {
+    text-align: center;
+}
+
+.team-item__photo {
+    height: 150px;
+    width: auto;
+    border-radius: 5px;
+}
+
+.team-item__namebox {
+    margin-top: 30px;
+}
+
+.team-item__namebox,
+.team-item__description {
+    color: #1c7aa7;
+    padding: 0 20px;
+}
+
+.team-item__namebox--function {
+    color: #418da7;
+}
+.team-item__description--links {
+    margin: 35px 0;
+}
+
+.team-item__description--link {
+    color: #1c7aa7;
+    text-decoration: none;
+    display: block;
+    margin-bottom: 10px;
+}
+
+.team-item__description--links svg {
+    font-size: 20px;
+    margin-right: 15px;
+}
+
+.team-item__description--link:hover {
+    color: #ffcc51;
+    text-decoration: none;
+    transform-origin: left;
+    transform: scale(1.2);
+    -webkit-transition: all 0.2s;
+    -moz-transition: all 0.2s;
+    -o-transition: all 0.2s;
+    transition: all 0.2s;
+}
+
+
+.social-media li:hover a {
```

Comment:
Try to keep files in order. Delete unused declaration and keep only one empty line between declarations.